### PR TITLE
Add Awesome Book html, css and Javascript

### PR DESCRIPTION
**Pull request by Diegogagan2587**
*Add plain html
*links to CSS but no styles yet
*add main.js
*add local storage within min.js
*set function to add and remove books from a books obj
*set function to display all elements from and obj to the html

### CODE REVIEW

**Review by abdallahmalima — COMMENTED** [review]
Hi @Diegogagan2587 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project that is not well-documented in the README file. It looks like you have taken the README file from your HTML capstone project and have not made significant changes to several sections. Please customize your README file to align with this project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: main.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+});
+const browserTitle = localStorage.getItem('title');
+const browserAuthor = localStorage.getItem('Author');
+if (browserTitle) {
+  bookTitle.value = browserTitle;
+}
+
+if (browserAuthor) {
+  bookAuthor.value = browserAuthor;
+}
```
Comment (gemmen29) [inline main.js:106]: - [ ] Good job on storing the value of the inputs field using local storage, but, there is no need for that. Kindly store the collection of books in the local storage instead as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md#:~:text=Make%20sure%20that%20data%20is%20preserved%20in%20the%20browser%27s%20memory%20by%20using%20localStorage.), to reload it after the page is refreshed.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi @Diegogagan2587, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. Well-implemented code 👍 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**